server/src/main/java/umm3601/Server.java

```diff
@@ -191,22 +191,21 @@ public static void main(String[] args) throws IOException, NoSuchAlgorithmExcept
 
         /// Deck and Card Endpoints ///////////////////////////
         /////////////////////////////////////////////
-        get("api/cards/:id", cardController::getCard);
-        get("api/cards", cardController::getCards);
-        get("api/decks", deckController::getDecks);
-        post("api/decks/add", deckController::addNewDeck);
-        get("api/decks/:id", deckController::getDeck);
-        post("api/decks/updateName", deckController::updateName);
-        post("api/cards/add", cardController::addNewCard);
-        post("api/addMany", cardController::addCardsToDeck);
-        post("api/deleteMany", cardController::deleteCardsFromDeck);
+//        get("api/cards/:id", cardController::getCard);
```

Comment:
Why are you commenting out code instead of deleting it? I think this applies to one of the other commits as well.

You really want to delete code like this, especially if the tests all pass and everything seems to work. Thanks to `git`, you can come back and get these later if you need to.

Reply:
Thanks for fixing this.